sdk/trace/src/main/scala/org/typelevel/otel4s/sdk/trace/SdkTracerProvider.scala

```diff
@@ -203,6 +203,10 @@ object SdkTracerProvider {
       copy(spanProcessors = this.spanProcessors :+ processor)
 
     def build: F[TracerProvider[F]] =
+      if (spanProcessors.isEmpty) Temporal[F].pure(TracerProvider.noop)
+      else create
+
+    private def create: F[TracerProvider[F]] =
```

Comment:
does this need to be its own method? should we just put its body in the `else` branch of the above method?